client/app/smallgroup/smallgroup.controller.js

```diff
@@ -115,13 +107,12 @@ angular.module('observatory3App')
   };
 
   var submitDayCode = function(code){
-      var user = Auth.getCurrentUser();
       $http.post('/api/attendance/attend', {
         dayCode: code
-      }).success(function(info){
+      }).success(function(){
```

Comment:
This is no different from the original code. Small groups do not currently require verification